src/hooks/useVenueScheduleEvents.ts

```diff
@@ -95,7 +96,11 @@ const useVenueScheduleEvents = ({
     [liveAndFutureEvents, firstRangeDateInSeconds]
   );
 
-  const endScheduleDate = sovereignVenue?.end_utc_seconds;
+  const endScheduleDate =
+    sovereignVenue?.end_utc_seconds &&
+    isFuture(fromUnixTime(sovereignVenue.end_utc_seconds)) &&
+    sovereignVenue?.end_utc_seconds;
```

Comment:
fixes the appearance of the weekdays' bar in the case when end_utc_seconds time has been already passed.

Reply:
my eyes might be betraying me, but it sure looks like a duplicate 
![image](https://user-images.githubusercontent.com/56254806/142375982-4ae98f18-0dbe-4826-a3bb-ec1667cffe18.png)

Reply:
there is logic. 🙃 
the meaning here is that if `sovereignVenue` and `sovereignVenue.end_utc_seconds` defined and it's in future, then `endScheduleDate = end_utc_seconds`, otherwise `undefined` or `false`.
But I agree, I should better do that explicitly. 

done here: https://github.com/sparkletown/sparkle/pull/2564/commits/38330a74973238bd651ff79fe21023785e3e8112